### ISSUE TITLE
Dynamic changing log level on ‘electricity delivered’ throws error

### ISSUE THREAD

**Issue by thomas-svrts** [ebaauw/homebridge-p1#48]
Changing the log level on ‘electricity’ works fine (and the changes are also applied for ‘gas’ and ‘electricity delivered’ logs)

But if you change it on ‘electricity delivered’ it causes a crash.

imo it shouldn’t be exposed on ‘electricity delivered’ (like it is for gas)

```

[21/3/2021 15:40:52] [P1] Electricity Delivered: error: TypeError: Cannot set property logLevel of #<Electricity> which has only a getter
    at CharacteristicDelegate.<anonymous> (/usr/lib/node_modules/homebridge-p1/lib/P1Service.js:72:28)
    at CharacteristicDelegate.emit (events.js:315:20)
    at CharacteristicDelegate.emit (/usr/lib/node_modules/homebridge-p1/node_modules/homebridge-lib/lib/Delegate.js:172:13)
    at CharacteristicDelegate._onSet (/usr/lib/node_modules/homebridge-p1/node_modules/homebridge-lib/lib/CharacteristicDelegate.js:391:10)
    at Characteristic.emit (events.js:315:20)
    at /usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1632:16
    at new Promise (<anonymous>)
    at Characteristic.<anonymous> (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1630:14)
    at step (/usr/lib/node_modules/homebridge/node_modules/tslib/tslib.js:143:27)
    at Object.next (/usr/lib/node_modules/homebridge/node_modules/tslib/tslib.js:124:57)

**Comment by ebaauw** [comment]
Hm, there shouldn't be a _Log Level_ in the _Electricity Delivered_ accessory; it should copy the log level from _Electricity_, as does the _Gas_ accessory.

**Comment by ebaauw** [comment]
Fix will be in next version, but is already in beta v1.2.1-1.

**Comment by thomas-svrts** [comment]
Tested and okay.

**Comment by ebaauw** [comment]
Released v1.2.1.
